src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep firstArg:'one', secondArg:'two'"));
```

Comment:
Oh, wait - so in practice, 
```groovy
monomorphStep([firstArg:'one', secondArg: 'two'])
```
and
```groovy
monomorphStep firstArg:'one', secondArg:'two'
```
Are effectively identical - they're both passing a `Map` to `monomorphStep`. I don't *like* the no-parens-at-all syntax, but it works (as should parens-but-not-brackets, which I'll test for). But if the general consensus is no-parens-at-all is the way to go, I'll tweak `Snippetizer` accordingly.